deploy/requirements/click.txt

```diff
@@ -22,4 +22,8 @@ polling2==0.5.0
 psycopg2-binary==2.9.9
 sqlalchemy==2.0.31
 pandas==2.2.2
-matplotlib==3.9.1
```

Comment:
I think we need additional requirement file for "scripts"
clic.txt is bad place for all of this